scripts/ci/run-vpp-tests.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+set -v
+set -e
+
+dir="$(dirname "$0")"
+TEMP_DIR=${TEMP_DIR:-/tmp/skydive-vpp}
+mkdir -p $TEMP_DIR/skydive-etcd
+
+vpp_create_loopback() {
+    loop=$(vppctl loopback create-interface | tr -d '\r')
+    ret=$?
+    if [ -z $loop ] ; then
```

Comment:
s/$loop/"$loop"/ - its' safer

Reply:
would change nothing in case of -z but I agree it's better practice (ex: -n test)

Reply:
ok